lua/ttt2/extensions/draw.lua

```diff
@@ -0,0 +1,436 @@
+---
+-- draw extension functions
+-- @author Mineotopia, LeBroomer, Alf21, saibotk
+
+AddCSLuaFile()
+
+if SERVER then return end
+
+local render = render
+local surface = surface
+local draw = draw
+local drawSimpleText = draw.SimpleText
+local table = table
+local cam = cam
+local tableCopy = table.Copy
+local mathRound = math.Round
+
+local colorShadowDark = Color(0, 0, 0, 220)
+local colorShadowBright = Color(0, 0, 0, 75)
+
+local function GetShadowColor(color)
+	local tmpCol = color.r + color.g + color.b > 200 and tableCopy(colorShadowDark) or tableCopy(colorShadowBright)
+
+	tmpCol.a = mathRound(tmpCol.a * (color.a / 255))
+
+	return tmpCol
+end
+
+---
+-- A function to draw an outlined box with a definable width
+-- @param number x The x position of the rectangle
+-- @param number y The y position of the rectangle
+-- @param number w The width of the rectangle
+-- @param number h The height of the rectangle
```

Comment:
box? rectangle? please choose one of the two or if you meant different things pls specify the relation of the two